recipes/cpp_project_framework/all/conanfile.py

```diff
@@ -0,0 +1,99 @@
+from conan import ConanFile
+from conans import CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class CppProjectFrameworkConan(ConanFile):
+    name = "cpp_project_framework"
+    license = "GNU Affero General Public License v3.0"
+    homepage = "https://github.com/sheepgrass/cpp_project_framework"
+    url = "https://github.com/conan-io/conan-center-index"  # Package recipe repository url here, for issues about the package
+    description = "C++ Project Framework is a framework for creating C++ project."
+    topics = ("c++", "project", "framework")
```

Comment:
```suggestion
    topics = ("project", "framework")
```

Reply:
I think `c++` is a highly related topic to the `cpp_project_framework`.  Is there any restriction to use it as a topic?

Reply:
No, but given Conan in the C++ package manager it's seems a little redundant 🤷 just my two cents

Reply:
better keep it from my point of view

Reply:
Keep `cpp` then. Need to remember that topics are keywords for searching, and search engines are trick. The `+` means multiple words on same tag.

Reply:
Keep cpp then. Need to remember that topics are keywords for searching, and search engines are trick. The + means multiple words on same tag.

Reply:
```suggestion
    topics = ("cpp", "project", "framework")
```

Reply:
Almost good to go, only need to update the topic.

Reply:
changed in latest version